CMakeLists.txt

```diff
@@ -53,15 +53,15 @@ if(NOT CMAKE_BUILD_TYPE)
   set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS Debug Release RelWithDebInfo MinSizeRel)
 endif()
 
-option(PRECICE_MPICommunication "Enables MPI-based communication and running coupling tests." ON)
-option(PRECICE_PETScMapping "Enable use of the PETSc linear algebra library." ON)
-option(PRECICE_PythonActions "Python support" ON)
-option(PRECICE_Packages "Configure package generation." ON)
+option(PRECICE_MPI_COMMUNICATION "Enables MPI-based communication and running coupling tests." ON)
+option(PRECICE_PETSC_MAPPING "Enable use of the PETSc linear algebra library for global-iterative RBF data mappings." ON)
+option(PRECICE_PYTHON_ACTIONS "Enable Python support for preCICE actions." ON)
+option(PRECICE_CONFIGURE_PACKAGE_GENERATION "Configure package generation." ON)
```

Comment:
Why the `CONFIGURE_` prefix here? Sounds inconsistent, as everything else should also be `CONFIGURE_*`. `PRECICE_PACKAGE_GENERATION` sounds enough right to me right now. The `CONFIGURE_` sounds better if we also group things with `FEATURE_`, `BINDINGS_`, etc.

Reply:
I was a bit afraid that the plain `PRECICE_PACKAGE_GENERATION` would indicate that cmake actually generates the packages for you, but I will change it back once we decided below

Reply:
> The CONFIGURE_ sounds better if we also group things with FEATURE_, BINDINGS_, etc.

Ok so do we keep it then?

Reply:
yes, makes sense.